### ISSUE TITLE
argmax axis=-1时结果不对

### ISSUE THREAD

**Issue by Meiyim** [PaddlePaddle/Paddle#16568]
为使您的问题得到快速解决，在建立Issues前，请您先通过如下方式搜索是否有相似问题:【搜索issue关键字】【使用labels筛选】【官方文档】

如果您没有查询到相似问题，为快速解决您的提问，建立issue时请提供如下细节信息：
- 标题：简洁、精准概括您的问题，例如“Insufficient Memory xxx" ”
- 版本、环境信息：
    1）PaddlePaddle版本：请提供您的PaddlePaddle版本号，例如1.1或CommitID
1.3.0
    2）CPU：预测若用CPU，请提供CPU型号，MKL/OpenBlas/MKLDNN/等数学库使用情况
    3）GPU：预测若用GPU，请提供GPU型号、CUDA和CUDNN版本号
GPU单卡
    4）系统环境：请您描述系统类型、版本，例如Mac OS 10.14，Python版本
- 训练信息
    1）单机/多机，单卡/多卡
    2）显存信息
    3）Operator信息
- 复现信息：如为报错，请给出复现环境、复现步骤
- 问题描述：请详细描述您的问题，同步贴出报错信息、日志、可复现的代码片段

```python
import numpy as np

import paddle
import paddle.fluid  as F
import paddle.fluid.layers  as L

train_program = F.Program()
startup_prog = F.Program()
with F.program_guard(train_program, startup_prog):
    with F.unique_name.guard():
        inputs = L.data(name='inputs', shape=[-1, -1], dtype='float32')
        loss = L.argmax(inputs, axis=-1)


place = F.CUDAPlace(0)
start_exe = F.Executor(place)
train_exe = F.Executor(place)

start_exe.run(startup_prog)

inputs_np = np.random.random([10, 2]).astype(np.float32)
print(paddle.__version__)
print(train_exe.run(program=train_program, fetch_list=[loss.name, inputs.name],
       feed={
            #'inputs': np.stack([np.array(range(10), dtype=np.float32)] * 10, axis=0),
            'inputs': inputs_np
        }))
print(np.argmax(inputs_np, axis=-1))
```
axis=-1有问题。为0或者None时符合预期。

运行结果
```shell
W0330 15:44:32.221108 175425 device_context.cc:263] Please NOTE: device: 0, CUDA Capability: 61, Driver API Version: 10.0, Runtime API Version: 9.0
W0330 15:44:32.221247 175425 device_context.cc:271] device: 0, cuDNN Version: 7.0.
W0330 15:44:32.221262 175425 device_context.cc:295] WARNING: device: 0. The installed Paddle is compiled with CUDNN 7.3, but CUDNN version in your machine is 7.0, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.
1.3.0
[array([ 8,  8,  8,  8,  8, 15, 15, 15, 19, 19], dtype=int64), array([[0.6415079 , 0.54418844],
       [0.7931793 , 0.99281013],
       [0.53737795, 0.5541438 ],
       [0.7883382 , 0.5702274 ],
       [0.9965097 , 0.22299178],
       [0.32447046, 0.51685405],
       [0.59805   , 0.48409164],
       [0.18593621, 0.785918  ],
       [0.62621224, 0.7022062 ],
       [0.4950174 , 0.76073635]], dtype=float32)]
[0 1 1 0 0 1 0 1 1 1]
```

**Comment by afterimagex** [comment]
框架不支持负数索引，concat也是同样错误

**Comment by Meiyim** [comment]
你们的文档用的就是-1做例子。
不支持也给个提示吧？编译时提示或者运行时提示都比稀里糊涂算一堆数字出来好，debug这个是很耗费精力的。
还有，这个axis的行为能不能统一一下，都支持负索引？

**Comment by afterimagex** [comment]
0.0.. 我也是用concat发现的这个问题。。
